### DISCUSSION TITLE
[GitHub]单独下载GitHub仓库中的某个文件夹，而不是全部

### DISCUSSION THREAD

**Discussion by tyn1998** [X-lab2017/open-wonderland#212] [1️⃣ 新鲜事]
# 用它

https://download-directory.github.io/

# 演示

Step1. 打开[这个](https://download-directory.github.io/)网址
Step2. 复制你想要下载的目录地址，例如：https://github.com/hypertrons/hypertrons-crx/tree/master/assets
Step3. 粘贴到输入框后回车

<img width="1328" alt="image" src="https://user-images.githubusercontent.com/32434520/200157007-91991bd9-1609-49b4-b23c-1f06d9665591.png">

Step4. 下载完毕后得到一个压缩包

<img width="860" alt="image" src="https://user-images.githubusercontent.com/32434520/200157053-ea403566-8670-4968-939c-86c21aefc130.png">

**Comment by will-ww** [comment]
这个相当实用呀~
